addons/portal/models/ir_qweb.py

```diff
@@ -14,10 +14,7 @@ def _prepare_frontend_environment(self, values):
         """
         irQweb = super()._prepare_frontend_environment(values)
         values.update(
-            is_html_empty=is_html_empty,
-            languages=lazy(lambda: [lang for
-                    lang in irQweb.env['res.lang'].get_available()
-                    if lang[0] in irQweb.env['ir.http']._get_frontend_langs()])
+            is_html_empty=is_html_empty
```

Comment:
Nitpicking: keep the trailing `,` to ease the diff should someone add another element later, or maybe even go with the `values['is_html_empty']` notation now that it's the only element